### DISCUSSION TITLE
Solar charging..

### DISCUSSION THREAD

**Discussion by zimbo86** [leeyuentuen/alfen_wallbox#158] [Q&A]
Hi, 
I dont really get the solar charging thing..
my purpose is to tell the Wallbox, with how many power to load.
Can I use solar level "comfort" and comfort level for this even when the wallbox is not connected to a solar smart meter?

**Comment by 2ktebn7ze5bh** [comment]
Hi @zimbo86, did you mange to make this work? I'm also looking for this setup to put the 'remaining' output of my solar panels as a maximum for the charger to load the car.

**Reply by zimbo86** [reply]
i switched over to evcc.. 
Spend 2$ a month and evcc handles everything for you..
in may i consumed 890 kWh, 430 kWh from these into the EV.
I imported 10 kWh from grid == 99% self sufficiency :)
